pt/cheatsheet-machine-learning-tips-and-tricks.md

```diff
@@ -1,285 +1,285 @@
 **1. Machine Learning tips and tricks cheatsheet**
 
-&#10230;
+&#10230; Dicas e Truques de Aprendizado de Máquina
```

Comment:
We've been using `Resumo`as a translation of `cheatsheet`. So I suggest this would be `Resumo de dicas e truques` and everything except the first letter should be lowercase